.github/workflows/post-submit.yaml

```diff
@@ -0,0 +1,74 @@
+name: Post Submit
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
+  pull-requests: read
+
+jobs:
+  push_to_registry:
+    name: Build and push images to quay.io/medik8s
+    runs-on: ubuntu-20.04
```

Comment:
NIT: There is a newer hosted runner image - https://github.com/medik8s/node-maintenance-operator/pull/73

Reply:
It can be applied also to the [`pres-submit.yaml`](https://github.com/medik8s/machine-deletion-remediation/blob/main/.github/workflows/pre-submit.yaml#L11) 😸